www/js/copy.js

```diff
@@ -100,73 +25,95 @@ function remapCeremony(unmapped) {
  */
 async function loadFromDB(id, idType) {
   let result;
+  pankteeArray.length = 0;
+
   if (idType === 'shabad') {
+    isCeremony = false;
     result = await baniDB.loadShabad(id);
-    remapShabad(result);
-  } else if (idType === 'bani') {
-    baniDB.loadBani(id, baniLengthMap[baniLength]).then(rows => {
-      remapBani(rows);
+    result.forEach(row => {
+      const remappedRow = global.controller.remapLine(row);
+      pankteeArray.push(remappedRow);
     });
-  } else if (idType === 'ceremony') {
-    baniDB.loadCeremony(id).then(rows => {
-      remapCeremony(rows);
+  } else {
+    isCeremony = idType === 'ceremony';
+    result = isCeremony ? baniDB.loadCeremony(id) : baniDB.loadBani(id, baniLengthMap[baniLength]);
```

Comment:
There's seems to be an off by 1. 

when mangal position is current saroops, sukhmani sahib is off by 1 line.

Reply:
yea im not sure about that. It only happens in sukhmani sahib.

Reply:
problem seems to be that its getting the preset mangal position, not the user prefered version.

Reply:
fixed